src/test/regress/expected/citus_local_tables_mx.out

```diff
@@ -724,5 +724,72 @@ $$);
  (localhost,57638,t,0)
 (2 rows)
 
+-- test adding foreign table to metadata with the guc
+SET citus.use_citus_managed_tables TO ON;
+CREATE TABLE foreign_table_test (id integer NOT NULL, data text);
```

Comment:
it seems like foreign tables could have serial/bigserial/sequence.nextval columns. Can you make sure undistribute/add node works fine with those?